app/controllers/experience_stages_controller.rb

```diff
@@ -5,6 +5,7 @@ def show
     @experience_stage = @challenge.experience_stage
     @themes = @experience_stage.themes
     @featured_experiences = @experience_stage.experiences.where(featured: true)
+    @order_field = params[:order_by].blank? ? '' : 'created_at DESC'
```

Comment:
Also, I'm guessing our version of "validation" here is essentially that when a user types in any arbitrary `order_by` param, we always will default to `created_at DESC` rather than the default order?